vyper/semantics/namespace.py

```diff
@@ -202,8 +202,6 @@ def validate_identifier(attr):
     "empty_bytes32",
     "max_int128",
     "min_int128",
-    "max_int256",
```

Comment:
why remove these two but not the others?

Reply:
these two values are not in `BUILTIN_CONSTANTS` in `vyper/ast/folding.py`.